### ISSUE TITLE
🐞 Node.JS : Directory.withNewFile escape error

### ISSUE THREAD

**Issue by zeroware** [dagger/dagger#3984]
### What is the issue?

It seems that the `contents` arguments of the `Directory.withNewFile` is not properly escaped

### Log output

Syntax Error GraphQL request (2:14762) Expected Name, found : [...]

### Steps to reproduce

```
   const stringWithQuotes = "This string has quotes \""
   const newFile = await client
       .container()
       .directory("/")
       .withNewFile("myfile", stringWithQuotes)
       .file("myfile").id()
```
  

### Dagger version

dagger node 0.1.1

### OS version

Linux 5.15

**Comment by aluzzardi** [comment]
I believe this has been fixed in 0.2.0. @slumbering ?

**Comment by slumbering** [comment]
New issue opened: #4130
cc @aluzzardi
